### PR TITLE
net-irc/irker: Version bump

### PR DESCRIPTION
**Pull request by bjacquin**
As per http://www.catb.org/~esr/irker/NEWS

Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by monsieurp** [comment]
@patricklauer

File: net-irc/irker/irker-2.18.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: Check py3.5 please

File: net-irc/irker/irker-2.18.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: Move to `EAPI=6`

File: net-irc/irker/irker-2.18.ebuild [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="socks5"
+
+DEPEND="app-text/docbook-xml-dtd:4.1.2
+	app-text/xmlto
+	socks5? ( dev-python/PySocks[${PYTHON_USEDEP}] )"
+
+src_prepare() {
+	# Rely on systemd eclass for systemd service install
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: Add `default` at the beginning

File: net-irc/irker/irker-2.18.ebuild [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	sed -i -e "/^SYSTEMDSYSTEMUNITDIR/d" Makefile \
+		|| die "sed failed"
+
+	# Prefix support
+	sed -i -e "/^ExecStart=/ s:=/:=${EROOT}:" irkerd.service \
+		|| die "sed failed"
+}
+
+src_install() {
+	emake DESTDIR="${ED}" install
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: this can also be replaced by `default`

File: net-irc/irker/irker-2.18.ebuild [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	# Not installed with the default Makefile
+	python_doscript irk irkerhook.py
+
+	newinitd "${FILESDIR}/irkerd.initd" irkerd
+	newconfd "${FILESDIR}/irkerd.confd" irkerd
+
+	systemd_dounit irkerd.service
+
+	dodoc NEWS README hacking.txt security.txt
+	dohtml irkerd.html irkerhook.html
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: deprecated, put into `HTML_DOCS`

**Comment by monsieurp** [comment]
@bjacquin ping

**Comment by bjacquin** [comment]
All of the comments have been fixed in updated PR.

File: net-irc/irker/irker-2.18.ebuild [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+	python_doscript "${ED}/usr/bin/irkerd"
+	# Not installed with the default Makefile
+	python_doscript irk irkerhook.py
+
+	newinitd "${FILESDIR}/irkerd.initd" irkerd
+	newconfd "${FILESDIR}/irkerd.confd" irkerd
+
+	systemd_dounit irkerd.service
+
+	dodoc NEWS README hacking.txt security.txt
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: put these in `DOCS` above `HTML_DOCS`

File: net-irc/irker/irker-2.18.ebuild [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+	# Prefix support
+	sed -i -e "/^ExecStart=/ s:=/:=${EROOT}:" irkerd.service \
+		|| die "sed failed"
+}
+
+src_install() {
+	default
+
+	python_doscript "${ED}/usr/bin/irkerd"
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: use `${ED%/}` to avoid double-slash

File: net-irc/irker/irker-2.18.ebuild [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+src_prepare() {
+	default
+
+	# Rely on systemd eclass for systemd service install
+	sed -i -e "/^SYSTEMDSYSTEMUNITDIR/d" Makefile \
+		|| die "sed failed"
+
+	# Prefix support
+	sed -i -e "/^ExecStart=/ s:=/:=${EROOT}:" irkerd.service \
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: you sure this shouldnt be `EPREFIX` instead of `EROOT`?

File: net-irc/irker/irker-2.18.ebuild [hunk 9]
```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_REQ_USE="ssl"
+
+inherit python-single-r1 systemd eutils
```
Comment (SoapGentoo) [inline net-irc/irker/irker-2.18.ebuild]: do you still need `eutils`?

**Comment by bjacquin** [comment]
All of the comments have been fixed in updated PR.

**Pull request closed without merging**